src/Simplex/Chat/Controller.hs

```diff
@@ -1076,6 +1078,10 @@ data ArchiveError
 -- | Host (mobile) side of transport to process remote commands and forward notifications
 data RemoteCtrlSession
   = RCSessionStarting
+  | RCSessionSearching
```

Comment:
RCSessionDiscovery?

Reply:
It was Discovery, but View displays it as "(searching)". Renamed to match other states in "-ing" form.